RibonDAO/core-api
# Feature/tech 249

# Description

- Adds a worker for sendind an email when users haven't donated in a week

## Does this pull request close any open issues?

- [#tech-249](https://ribon.atlassian.net/browse/TECH-249?atlOrigin=eyJpIjoiYTNlZThkMmM4NDM2NGRlYmE1ODk4MTNmZDAwODFjNzEiLCJwIjoiaiJ9)

## Type of change


## How to test

- Run sidekiq and call the job for a user (with your e-mail) whose last donation was a week ago (expect to receive the e-mail in your inbox)
